### DISCUSSION TITLE
Referente a la tarea asignada el día de hoy

### DISCUSSION THREAD

**Discussion by zalgodxx** [hackatomik/SISCOx256#37] [Repo Questions]
La verdad es que leí la tarea pero no entendí nada de lo que hay que hacer y la tarea cayó de repente. Si podrían explicar un poco más de qué va el trabajo que hay que entregar hoy, se los agradeceria por favor 

 
![30ec0541-1e76-40f0-80a7-ba3defae7ff0](https://github.com/user-attachments/assets/3866cdbe-3fea-46af-9833-0136261b16bc)

**Comment by quantumquirkxyz** [comment]
Bueno, la tarea en si, es solo para avisar que puedes usar esta misma plataforma (Discussions) para preguntar cosas como estas, ademas, lo ideal es que hubieras desarrollado su knock knock server con sus tribus para ver que tal les iba, si no pueden no hay problema
